echo-plugins-test/echo-plugins-test.gradle

```diff
@@ -0,0 +1,29 @@
+apply from: "${project.rootDir}/gradle/kotlin.gradle"
```

Comment:
I thought we were going to move these to `{app}-api-test`?

Reply:
I was thinking we'd have `{app}-api`, `{app}-api-tck`, and `{app}-plugins-test`.  I guess the last one could be called `{app}-api-test`. This module is really about loading actual plugins into the app, which is why I think it makes sense to have `plugins` in the name. 🤔

Reply:
Also, this is following the same pattern I have put in place in Orca and Gate.